src/tests/JIT/Regression/JitBlue/GitHub_27678/GitHub_27678.ilproj

```diff
@@ -2,6 +2,12 @@
   <PropertyGroup>
     <DebugType>None</DebugType>
     <Optimize>True</Optimize>
+
+    <!-- * Assertion at
+         /__w/1/s/src/mono/mono/mini/memory-access.c:120, condition `size < MAX_INLINE_COPY_SIZE' not met -->
+    <MonoAotIncompatible>true</MonoAotIncompatible>
```

Comment:
Why do we need to mark the test as MonoAOT-incompatible? I would expect that just by using the <core>RequiresProcessIsolation</code> you should fix the build error as the test script will never be run (having been filtered out by the issues.targets-based exclusion list for the merged wrapper); adding the <code>MonoAotIncompatible</code> property seems to indicate that this test can basically never run under MonoAOT by design and I don't see why it should apply here - once the MonoAOT bug has been fixed and the issues.targets exclusion removed, the test should start running again; with this annotation it will remain buried for eternity until someone happens to notice it and / or manually audit such annotations.

Reply:
(side note - appears to be moot for this PR given the fix)

That's a good point. I've so used to adding RequiresProcessIsolation for existing tags like GCStressIncompatible that I followed that here - perhaps considering the MonoAOTIncompatible to be a bit of documentation. But you're right and that documentation can simply be a comment. It would probably make sense to add a comment in issues.targets to remove the RPI as well. I'll do that for future issues (or here if the PR falls through for some reason).  Thanks!